ci/requirements-2.7_LOCALE.pip

```diff
@@ -1 +1,3 @@
 blosc
+httplib2
+google-api-python-client == 1.2
```

Comment:
you don't need to have this work in EVERY version. In fact let's just care about 2.7 (where you specify exact versions, e.g. the 1.2 client) and 3.4 (where you don't, e.g. the latest)